zkevm-circuits/src/circuit_tools/constraint_builder.rs

```diff
@@ -1133,6 +1155,34 @@ macro_rules! circuit {
             }};
         }
 
+        #[allow(unused_macros)]
+        macro_rules! cur {
+            ($column:expr) => {{
+                $column.query($meta, Rotation::cur())
+            }};
+        }
+
+        #[allow(unused_macros)]
+        macro_rules! next {
+            ($column:expr) => {{
+                $column.query($meta, Rotation::next())
+            }};
+        }
+
+        #[allow(unused_macros)]
+        macro_rules! prev {
+            ($column:expr) => {{
+                $column.query($meta, Rotation::prev())
+            }};
+        }
+
+        #[allow(unused_macros)]
+        macro_rules! query {
+            ($column:expr, $rot:expr) => {{
+                $column.query($meta, Rotation($rot as i32))
+            }};
+        }
```

Comment:
If the `meta` dependency is gone for querying cells, do you still think we need these macro's or just using the `Querier` syntax directly also works for you?

Reply:
Do you mean that column.cur() would work without the meta? In that case the macros would be unnecessary I believe.

Reply:
Yep!